src/toxcore/dht/daemon_state.rs

```diff
@@ -59,9 +59,10 @@ impl DaemonState {
 
         let mut request_queue = server.request_queue.write();
         let nodes_sender = nodes.iter()
-            .map(|node|
-                server.send_nodes_req(&mut request_queue, &(*node).into(), server.pk)
-            );
+            .map(|node| {
+                let ping_id = request_queue.new_ping_id(node.pk);
```

Comment:
This line is redundant. Is there any reason for avoiding to pass request_queue as an argument?

Reply:
I made it to be consistent with `send_ping_req` but you are right - changed both to taking `request_queue`.